README.md

```diff
@@ -93,116 +91,54 @@ $ php composer.phar create-project elcodi/bamboo bamboo -sdev
 > The installation process will ask you for some parameters like the database
 > driver, username, password, database name, etc
 
-Enter your directory to start the configuration step
+Enter your directory to start the configuration step and use our magic command
+to create a complete development environment. Just one single line to rule them
+all
 
 ``` bash
 $ cd bamboo/
+$ php app/console elcodi:install
 ```
 
-### 2. Init your database
-
-Now we should create the database and all the application schema. Symfony
-provides you an easy way for doing that.
-
-``` bash
-$ php app/console doctrine:database:create
-$ php app/console doctrine:schema:create
-```
-
-We also load some fixtures to show on our store. This fixtures will set your
-store in a testing mode, with some categories, some manufacturers and a bunch of
-t-shirts. Only for testing purposes :)
+By default only Spain will be installed. We have focused this step to run as 
```

Comment:
Spain -> Locations

This has nothing to do with the language. By default, Bamboo is (and will be) in english :)